src/Illuminate/Filesystem/Filesystem.php

```diff
@@ -606,4 +607,35 @@ public function cleanDirectory($directory)
     {
         return $this->deleteDirectory($directory, true);
     }
+
+    /**
+     * Compute the relative path to a target from a given base path.
+     *
+     * @param  string  $target
+     * @param  string  $basePath
+     * @return string
+     */
+    public function relativePath($target, $basePath)
```

Comment:
Shouldn't we also handle windows style paths?

Reply:
Hence the initial usage of `webmozart/path-util`...